hicommonwealth/commonwealth
# Added integration section for stakes in community management

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6680

## Description of Changes
- Added integration section for stakes in community management for community admins

## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_EXISTING_COMMUNITY_STAKE_INTEGRATION_ENABLED` in `.env`
- Visit `/:scope/manage/integrations` page of any community (with and without stake enabled) you are an admin of, verify you see the stake section with relevant UI depending on stake enablement, verify that clicking the button redirects you to `/:scope/manage/integrations/stake`
- Visit `/:scope/manage/integrations` page of any community that cannot have stake enabled yet (any chain with ethChainId not included in [these](https://github.com/hicommonwealth/commonwealth/blob/78ecc499fa5d3b83621b494cc5873e9907fb4759/libs/core/src/commonProtocol/chainConfig.ts#L2-L7)), verify you see the disabled action button for stake enablement with `Stake is not supported on your network` tooltip

## Deployment Plan
N/A

## Other Considerations
N/A
